lib/messageUtils.ts

```diff
@@ -105,14 +105,15 @@ export function getDashboard(
   branchSlug?: string,
   startDate?: string | null,
   endDate?: string | null,
+  isCompleted?: boolean,
```

Comment:
If you could add a comment explaining that this could be useful for messages that used to be in remote settings or old versions of Firefox, that would be nice, because it took me a bit to figure this out, and I wouldn't want the code to be inadvertently removed because of lack of context.